_docs/installation/partitioning/encrypted.md

```diff
@@ -251,6 +251,25 @@ When doing root on ZFS, LUKS does not influence the `root=`. You just have to
 specify something like `root=ZFS=mypool/root/whatever` and the initramfs will
 take care of the rest, provided the `crypttab` mappings are correctly set up.
 
+As `cryptsetup` currently does not detect the root filesystem when it's on ZFS
```

Comment:
cryptsetup doesn't deal with root filesystem at all (the initramfs hooks that deal with cryptsetup integration do), so this wording is confusing

Reply:
Agree, I rephrased that, hope it's ok.